doc/ref_impl/cntt-ri2/chapters/chapter03.md

```diff
@@ -72,22 +72,23 @@ The network fabric shall comprise redundant layer 2 connectivity between hosts.
 The requirements on infrastructure hardware used to deploy and run the RI2 are as follows:
 
 1. One (1) physical or virtual server dedicated as a jump / test host
-1. Six (6) physical or virtual servers, serving as either master (controller) or compute/worker nodes
+1. Five (5) physical or virtual servers, serving as either master (controller) or compute/worker nodes
 
 The jump / test host allows to install and manage the operation of the RI2. For example, the host can be used to execute the installer software to deploy and configure the RI2 on the other physical or virtual servers. Once installed, the host may be used to run test tools or it may be used to generate test traffic intended for performance measurements. To support the latter, the server provided for the jump / test host shall meet similar hardware requirements as the master (controller) / compute hosts. If the jump host is used only for deployment or execution of functional tests after deployment, a VM with the resources as outlined below is sufficient.
 
 Each server shall meet the following minimum specifications:
 
 * CPU
-  *  2x x86_64 CPU sockets, providing 20 cores each, 40 simultaneous multi-threads (SMT), at 2.2 GHz
+  *  2x x86_64 CPU sockets, providing 20 cores each, 40 simultaneous multi-threads (SMT), at 2.2 GHz. If using VM's, the servers must be equipped with at least 16 vCPUs.
 * Memory
-  * 256 GB RAM
+  * 32 GB RAM
```

Comment:
The 32 GB RAM, 320 GB disk etc are the maximum assigned to a Pod. So is the intent to make the max pod ask to be a minimum for the Server?

Reply:
The [ RA2 documentation](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/kubernetes/chapters/chapter04.md) specifies "at least" 32GB RAM/320 GB disk for the worker nodes so I assumed that it refers to the minimum requirement of the worker nodes (assuming that node=server in our 5 server k8s setup). Is there any formal definition of Nodes/Pods/Servers within the Anuket documentation?

Reply:
I assume that the 32GB/320GB doesn't count for the workloads or additional components in a specific RI deployment such as logging and monitoring? 

@pgoyal01 I am not sure I follow what do you mean by "assigned to a pod", do you mean all the k8s controller pods together should be assignedto at least 32GB?

Reply:
@jgu17  What I am trying to state, not very clearly as usual, is that the physical server has to be sized larger than the specification below (and it is stated in the specs also).  BTW if the server only supports 32 GB then that is the total available for OS, monitoring, tools, utilities, nodes, etc.

From RA2 [4.2 Kubernetes Node](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/kubernetes/chapters/chapter04.md#42-kubernetes-node) (emphasis mine):

Ref | Specification | Details | Requirement Trace | Reference Implementation Trace
-- | -- | -- | -- | --
ra2.ch.010 | Local Filesystem Storage Quantity | The Kubernetes Nodes must be equipped with local filesystem capacity  of **at least 320GB** for unpacking and executing containers. Note, _extra  should be provisioned_ to cater for any overhead required by the  Operating System and any required OS processes such as the container  runtime, Kubernetes agents, etc. | e.cap.003 |  
ra2.ch.012 | Kubernetes Node RAM Quantity | The Kubernetes Nodes must be equipped with **at least 32GB of RAM**.  Note, _extra should be provisioned_ to cater for any overhead required by  the Operating System and any required OS processes such as the container  runtime, Kubernetes agents, etc. | e.cap.002 |  

Reply:
32 GB RAM, 320 GB disk seems really low for a lab machine. I think there should be distinction between minimum called for by the spec vs what is practical for a lab environment to provide flexibility needed for dev/testing environment

Reply:
Agree with Trevor and Lincoln on these specifications for RAM and Storage. Fitting-out a Lab installation with minimums that are so low (32/320) means replacing that system when the minimums rise in a short amount of time, and there are many such systems in the lab. Lincoln's (256/1024)G suggestion has some legs for the future.